src/main/webapp/gallery.html

```diff
@@ -11,21 +11,80 @@
   </head>
   
-  <body>
+  <body onload="">
     <div id="navigation"></div>
 
     <!--------------- HEADER ---------------->
     <header class="container-fluid">
-      <h1>GALLERY</h1>
+      <div class="gallery-heading">
+        <h1>GALLERY</h1>
+      </div>
     </header>
 
     <!--------------- CONTENT ---------------->
-    <div id="content">
-      <!-- Page Content -->
-      <div id="gallery-page" class="container-fluid">
-        <div class="grid bg-dark">
-          <div class="grid-sizer"></div>
+    <div class="container-fluid">
+      <div class="row">
+        <nav class="col-md-2 d-none d-md-block bg-light sidebar">
+          <div class="sidebar-sticky">
+            <h2 class="sidebar-heading d-flex justify-content-between align-items-center mt-4 mb-1">
+              <span>Filters</span>
+            </h2>
+            <!-- <h6 class="sidebar-heading d-flex justify-content-between align-items-center px-3 mt-4 mb-1 text-muted">
+              <span>Kingdom</span>
+            </h6>
+            <ul class="nav flex-column mb-2">
+              <li class="nav-item">
+                <a class="nav-link" href="#">
+                  Carnivora
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">
+                  Carnivora
+                </a>
+              </li>
+            </ul> -->
+
+            <h6 class="sidebar-heading d-flex justify-content-between align-items-center px-3 mt-4 mb-1 text-muted">
+              <span>Status</span>
+            </h6>
+            <ul class="nav flex-column mb-2">
+              <li class="nav-item">
+                <button class="btn active" onclick="filterSelection('all')"> Show all</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('CR')"> Critically Endangered</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('EN')"> Endangered</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('VU')"> Vulnerable</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('NT')"> Near Threatened</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('LC')"> Least Concern</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('DO')"> Domesticated</button>
+              </li>
+              <li class="nav-item">
+                <button class="btn" onclick="filterSelection('DD')"> Data Deficient</button>
+              </li>
+            </ul>
+          </div>
+        </nav>
+
+        <div class="col-md-9 ml-sm-auto col-lg-10 px-4">
```

Comment:
What do the classes here do? I assumed masonry took care of the columns.

Reply:
This is bootstrap's inbuilt grid system! It's basically to create the sidebar and separate it from the gallery grid